src/hooks/Transactor.ts

```diff
@@ -127,9 +94,13 @@ export function useTransactor({
       try {
         let result
 
+        const txTitle = options?.title ?? functionName
+
         if (tx instanceof Promise) {
           console.info('AWAITING TX', tx)
           result = await tx
+
+          addTransaction?.(txTitle, result)
```

Comment:
each time the transactor creates a transaction, we beam it to the TxHistoryProvider via TxHistoryContext

Reply:
<3